pyServer/manifests/linux/sql-iaas

```diff
@@ -0,0 +1,35 @@
+echo,### Probing Directories ###
```

Comment:
Can we add "noscan" after the files for which we are sure that those files not going to contain any customer's secrets

Reply:
added noscan to files which does not contains any secrets.